app/panel/page.tsx

```diff
@@ -1,21 +1,7 @@
 import { Container, Button, Row } from "@yakad/ui";
 import Link from "next/link";
-import { cookies } from "next/headers";
-import { redirect } from "next/navigation";
-import { getUserProfile, UserProfile } from "./profile/profile";
 
 export default async function Page() {
-    const cookie = cookies();
```

Comment:
What has replaced cookies?

Reply:
We didn't need cookies in this page, so I removed it, previously we were getting the user profile in this page but now it's moved to account page.